modules/sdk-core/src/bitgo/wallet/wallets.ts

```diff
@@ -95,9 +95,15 @@ export class Wallets implements IWallets {
    *    "n": number of keys available on the wallet (3)
    *    "keys": array of keychain ids
    */
-  async add(params: AddWalletOptions = {}): Promise<any> {
+  async add(params: AddWalletOptions): Promise<any> {
+    params = params || {};
```

Comment:
nit: not sure if this is necessary since `param` is already declared non null in the function param

Reply:
Without it, line 103 throws an typescript error

Reply:
I did make some changes after that. Lemme confirm.